src/sass/Abstracts/_Mixins.scss

```
@@ -0,0 +1 @@
+// Mixins: used for styles that you want to reuse
```

```suggestion
// Mixins: used for styles that you want to reuse.
```

Nit: missing period.